examples/ex-batched-matrix-multiply.cpp

```diff
@@ -142,15 +143,15 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 //
 // Allocate space for data in layout 2
 //
-  double *A2 = memoryManager::allocate<double>(NCOLS * NROWS * N);
-  double *B2 = memoryManager::allocate<double>(NCOLS * NROWS * N);
-  double *C2 = memoryManager::allocate<double>(NCOLS * NROWS * N);
+  double *A2 = memoryManager::allocate<double>(N_c * N_r * N);
+  double *B2 = memoryManager::allocate<double>(N_c * N_r * N);
+  double *C2 = memoryManager::allocate<double>(N_c * N_r * N);
 
 //
 // Permuted layout - equivalent to indexing via
-// A[e + N*(r + NROWS*c)]
+// A[e + N*(c + N_c*r)]
```

Comment:
Better to make the indexing description look the same as the one above (line 132)